run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Submit the pipeline as a background process with ./run.sh
+# module load R # Uncomment if R is an environment module.
+nohup nice -4 R CMD BATCH run.R &
```

Comment:
Are you running this from shell? I ask out of curiosity - I'm pretty new to working with shell scripts.

Reply:
No. I think it was included with the pull because I didn't add it.